doc/source/building/working_with_images/vmx_setup_for_ec2.rst

```diff
@@ -0,0 +1,129 @@
+.. _setup_for_ec2:
+
+KIWI image description for Amazon EC2
+=====================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   vmx images built with KIWI and references the following
+   articles:
+
+   * :ref:`vmx`
+
+A virtual disk image which is able to boot in the Amazon EC2
+cloud framework has to comply the following constraints:
+
+* Xen tools and libraries must be installed
+* cloud-init must be installed
+* cloud-init configuration for amazon must be provided
```

Comment:
Is "cloud-init" a command? A package? Please add this to the sentence like "cloud-init package must be installed"